ui/lib/core/addon/components/string-list.hbs

```diff
@@ -53,15 +53,15 @@
         {{/if}}
       </div>
       {{#if (includes index this.indicesWithComma)}}
-        <Icon class="is-flex-v-centered has-text-highlight" @name="alert-triangle-fill" />
+        <Hds::Alert class="is-flex-v-centered" @type="compact" @color="warning" />
```

Comment:
If we're using compact here, should we use AlertInline instead?

Reply:
This is just so the icon + color to the left of the input matches the alert below, there's no message here.

Reply:
updated screenshot in comment below